fuel/transformers/__init__.py

```diff
@@ -200,31 +325,28 @@ def __init__(self, data_stream, dtype, **kwargs):
         if dtype == 'floatX':
             dtype = config.floatX
         self.dtype = dtype
-        super(Cast, self).__init__(data_stream, **kwargs)
+        kwargs.setdefault('axis_labels', data_stream.axis_labels)
```

Comment:
Same question. Seems like an even clearer case where we can force `axis_labels` in the arg list below. I guess we can   argue "caveat emptor" or something but allowing the constructor to do something nonsensical doesn't seem worthwhile.